css/theme.less

```diff
@@ -125,7 +125,7 @@ body:before {
     position: relative;
     padding: 0;
     margin: 0 0 20px;
-    border-bottom: 15px double @ini_theme_color_alt;
+    border-bottom: 0px solid @ini_theme_color_alt;
```

Comment:
Minor: If you want to remove the border it's better to remove the whole line. Otherwise it's dead code.

Reply:
You are absolutely right. Like I said, I simply transfered my userall.css. So in that case I had to negate the styling. I should definitely go through to find other remnants like this.